src/components/partner/dto/update-partner.dto.ts

```diff
@@ -53,6 +53,10 @@ export abstract class UpdatePartner {
   @IsId({ each: true })
   @Transform(({ value }) => (value ? uniq(value) : undefined))
   readonly fieldRegions?: ReadonlyArray<IdOf<FieldRegion>>;
+
+  @Field(() => [IDType], { name: 'languagesOfConsulting', nullable: true })
+  @Transform(({ value }) => (value ? uniq(value) : undefined))
+  readonly languagesOfConsulting?: ReadonlyArray<IdOf<Language>> = [];
```

Comment:
We want to assume a missing prop here means unchanged, not clear existing value.
```suggestion
  readonly languagesOfConsulting?: ReadonlyArray<IdOf<Language>>;
```

Reply:
Good catch... I think I copied this from the list dto and didn't think of the implication here